spec/acceptance/suites/default/87_disable_fips_spec.rb

```diff
@@ -69,13 +69,12 @@
         retry_on(agent, 'uptime', :retry_interval => 15 )
 
         retry_on(agent, 'puppet agent -t',
-          :desired_exit_codes => [0],
+          :desired_exit_codes => [0,2],
```

Comment:
Why was this changed to allow exit code 2 (changes detected)?  Shouldn't we be testing that it converges to a stable configuration?

Reply:
It's not an idempotency check and the reboot notification may trigger an exit code of  `2`